recipes/lora_finetune_distributed.py

```diff
@@ -287,6 +287,10 @@ def _setup_model(
                 f"Model instantiation took {time.perf_counter() - init_start:.2f} secs"
             )
 
+            cfg_model.apply_lora_to_output = getattr(
+                cfg_model, "apply_lora_to_output", False
+            )
```

Comment:
Technically you can probably just set this to `apply_lora_to_output` and not modify `cfg_model`, but not a huge deal either way

Reply:
Thank you for your suggestion, I update them in `recipes/lora_finetune_distributed.py` and `recipes/lora_finetune_single_device.py`.